modules/media/jsx/editFormIndex.js

```diff
@@ -0,0 +1,13 @@
+/* global ReactDOM */
+
+import MediaEditForm from './editForm';
+const args = QueryString.get(document.currentScript.src);
+
+$(function() {
+  var mediaEditForm = <MediaEditForm
+    DataURL={`${loris.BaseURL}/media/ajax/FileUpload.php?action=getData&idMediaFile=${args.idMediaFile}`}
```

Comment:
Edit page breaks now. I believe this should be `args.id` since it's the `id` that is passed in the URL